src/jit/lsra.cpp

```diff
@@ -3391,8 +3391,9 @@ class LocationInfoListNodePool final
     {
         if (preallocate > 0)
         {
-            size_t preallocateSize   = sizeof(LocationInfoListNode) * preallocate;
-            auto*  preallocatedNodes = reinterpret_cast<LocationInfoListNode*>(compiler->compGetMem(preallocateSize));
+            size_t preallocateSize = sizeof(LocationInfoListNode) * preallocate;
+            auto*  preallocatedNodes =
+                reinterpret_cast<LocationInfoListNode*>(compiler->compGetMem(preallocateSize, CMK_LSRA));
```

Comment:
Using both auto and reinterpret_cast on the same line seen sketchy to me.

Can we just declare the type as `LocationInfoListNode*`  here instead of using auto

Reply:
I'm happy to do that; I'm also a favor of stronger typing :-)